### DISCUSSION TITLE
Installation proble

### DISCUSSION THREAD

**Discussion by medi0305** [boxbilling/boxbilling#1117] [Q&A]
![20211112_142916](https://user-images.githubusercontent.com/56176516/141475266-1c8e3ada-487a-4b7d-99ac-9e0e3b2e7c5f.jpg)

I have a problems in installing Boxbilling on Ubuntu serter 20.04. LTS.

There is mesage as per attached picture.

Can somebody help me tpo solve issue.
Thanks in advance

Dejan

**Comment by BelleNottelling** [comment]
Using docker? I have no idea if that is supposed to work at all, I have never used it and I don't know of anyone who has used it
